pennylane/circuit_graph.py

```diff
@@ -100,12 +100,12 @@ class CircuitGraph:
 
     Args:
         ops (Iterable[Operator]): quantum operators constituting the circuit, in temporal order
-        variable_deps (dict[int, list[ParameterDependency]]): Free parameters of the quantum circuit.
-            The dictionary key is the parameter index.
+        primary_deps (dict[int, list[ParameterDependency]]): Mapping from flattened primary
+            parameter indices to Operators that depend on them.
     """
 
-    def __init__(self, ops, variable_deps):
-        self.variable_deps = variable_deps
+    def __init__(self, ops, primary_deps):
+        self.primary_deps = primary_deps
```

Comment:
`primary_deps` records which Operators depend on which primary parameters, it's mostly used in differentiation. I did not want to use the words `variable` or `free parameter` since they're really not used anywhere else for this purpose. Maybe `primary_parameter_deps` or `primary_par_deps`?

Reply:
`primary_par_deps` sounds good!

Reply:
I changed it.